xarray_sentinel/sentinel1.py

```diff
@@ -100,23 +101,17 @@ def open_orbit_dataset(product_path: str) -> xr.Dataset:
     shape = len(orbit)
 
     reference_system = orbit[0]["frame"]
-    data_vars: T.Dict[str, T.List[T.Any]] = {
-        "time": [],
-        "x": [],
-        "y": [],
-        "z": [],
-        "vx": [],
-        "vy": [],
-        "vz": [],
-    }
+    variables = ["time", "x", "y", "z", "vx", "vy", "vz"]
+    data_vars: T.Dict[str, T.List[T.Any]] = {var: ("time", []) for var in variables}  # type: ignore
+
     for k in range(shape):
-        data_vars["time"].append(orbit[k]["time"])
-        data_vars["x"].append(orbit[k]["position"]["x"])
-        data_vars["y"].append(orbit[k]["position"]["y"])
-        data_vars["z"].append(orbit[k]["position"]["z"])
-        data_vars["x"].append(orbit[k]["velocity"]["x"])
-        data_vars["y"].append(orbit[k]["velocity"]["y"])
-        data_vars["z"].append(orbit[k]["velocity"]["z"])
+        data_vars["time"][1].append(orbit[k]["time"])
+        data_vars["x"][1].append(orbit[k]["position"]["x"])
+        data_vars["y"][1].append(orbit[k]["position"]["y"])
+        data_vars["z"][1].append(orbit[k]["position"]["z"])
+        data_vars["vx"][1].append(orbit[k]["velocity"]["x"])
```

Comment:
Minor: `vx` is short and pretty obvious, but less explicit than `velocity_x`. Not sure which of the two is best.

Reply:
`velocity_x` is too long for me. but I should add it in the long name

Reply:
added in long name